tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestQueryOnSelfDescTable.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.tajo.engine.query;
+
+import org.apache.tajo.QueryTestCaseBase;
+import org.apache.tajo.exception.AmbiguousColumnException;
+import org.apache.tajo.exception.TajoException;
+import org.junit.After;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class TestQueryOnSelfDescTable extends QueryTestCaseBase {
+
+  public TestQueryOnSelfDescTable() throws IOException, TajoException, SQLException {
+    super();
+
+    executeString(String.format("create external table self_desc_table1 (*) using json location '%s'",
+        getDataSetFile("sample1"))).close();
+
+    executeString(String.format("create external table self_desc_table2 (*) using json location '%s'",
+        getDataSetFile("sample2"))).close();
+
+    executeString(String.format("create external table self_desc_table3 (*) using json location '%s'",
+        getDataSetFile("tweets"))).close();
+
+    executeString(String.format("create external table github (*) using json location '%s'",
+        getDataSetFile("github"))).close();
+  }
+
+  @After
+  public void teardown() throws TajoException, SQLException {
+    executeString("drop table self_desc_table1").close();
```

Comment:
It's a trivial comment. It would be better if you put ``IF NOT EXISTS`` to them. It will suppress error messages in unexpected cases.